09-redux/classworks/Volodkova_Anna/src/resourses/types.js

```diff
@@ -0,0 +1,4 @@
+export const SET_REACT_SUBREDDITS_POSTS = 'REACT_SUBREDDITS_POSTS';
+export const SET_SEARCH_VALUE = 'GET_SEARCH_VALUE';
```

Comment:
const SET_SEARCH_VALUE = 'GET_SEARCH_VALUE'
does anything bother you?

Reply:
Doesn't